parsons/mobilize_america/ma.py

```diff
@@ -135,9 +149,11 @@ def get_events(self, organization_id=None, updated_since=None, timeslot_start=No
                 This is helpful in situations where you have a regular sync
                 running and want to ensure that the column headers remain static.
 
+                If ``max_timeslots`` is 0, no timeslot columns will be included.
+
         `Returns`
-            Parsons Table or dict or Parsons Tables
-                See :ref:`parsons-table` for output options.
+            Parsons Table or dict
+                See :ref:`timeslot_table` for output options.
```

Comment:
Oh, I was getting my doc formatting wrong. I think I meant to use `` to refer back to the function parameter, since that changes the output options. TBH I was making a bit of a sneaky change anyway since I was changing the meaning of the sentence without really changing the wording.

I can revert it to the "see Parsons table" language. (Personally I think it's unnecessary to write out the return type and then say "see the docs on the return type for output options" but it's fine.)

Reply:
I changed it so that the `parsons.Table`s in the return docstring link to the "Parsons Table" section